log4j-api-test/src/main/java/org/apache/logging/log4j/test/junit/TempLoggingDirectory.java

```diff
@@ -78,7 +79,8 @@ public void beforeEach(ExtensionContext context) throws Exception {
         final Object instance = context.getRequiredTestInstance();
         for (final Field field : fields) {
             if (loggingPath != null) {
-                StatusLogger.getLogger().warn("Multiple fields with @TempLoggingDir annotation are not supported.");
+                StatusLogger.getLogger()
+                        .warn("Multiple instance fields with @TempLoggingDir annotation are not supported.");
```

Comment:
Again, why not?

Reply:
Same as above: do we need to define multiple logging paths per test?
Which one would be used to resolve to `${test:logging.path}`?